pervane/serve.py

```diff
@@ -55,6 +55,8 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 from werkzeug.utils import secure_filename
 
+from pathlib import Path
```

Comment:
let's not directly use class, instead let's use it from the module.
import pathlib
...
pathlib.Path